types/crpc/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"no-unnecessary-generics": false
```

Comment:
That `TRes` is really just a way of achieving a type assertion. I'd recommend making the return type `Promise<unknown>`so users are aware of the explicit cast.

Reply:
The intent of this function is to retrieve the JSON payload from an API and assert it to the type provided. So this behaviour isn't unexpected. Removing TReq, and replacing with `any` does make sense, but I think it goes against the grain to do unknown, then do the type assertion on the client side (as I feel this just adds more code).